packages/contracts/contracts/Artonomous.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity ^0.4.23;
```

Comment:
Any reason to not use the latest solidity version 0.4.24?

Reply:
no reason. 0.4.23 was what the Migrations contract was using so was just following convention. I'll update them all though.